dev/setup_darwin.go

```diff
@@ -202,18 +202,30 @@ func InstallIntoSystem(config *InstallIntoSystemArgs) error {
 	return nil
 }
 
-func Uninstall(domains []string) {
-	plist := homedir.MustExpand("~/Library/LaunchAgents/io.puma.dev.plist")
+func Uninstall(launchAgentDirPath string, domains []string) func() {
+	// Default: ~/Library/LaunchAgents/
+	plistDir := homedir.MustExpand(launchAgentDirPath)
+	plist := filepath.Join(plistDir, "io.puma.dev.plist")
 
-	// Unload a previous one if need be.
+	// Remove puma-dev daemon
 	exec.Command("launchctl", "unload", plist).Run()
-
 	os.Remove(plist)
-
 	fmt.Printf("* Removed puma-dev from automatically running\n")
 
 	for _, d := range domains {
 		os.Remove(filepath.Join("/etc/resolver", d))
 		fmt.Printf("* Removed domain '%s'\n", d)
 	}
+
+	// Remove all `Puma-dev CA` certificate entries from macOS keychain
+	if err := DeleteAllPumaDevCAFromDefaultKeychain(); err != nil {
+		fmt.Printf("! Unable to remove all Puma-dev CA certs from macOS keychain: %s\n", err)
+		return func() {}
+	} else {
+		fmt.Printf("* Removed all Puma-dev CA certs from macOS keychain.\n")
+
+		return func() {
+			os.RemoveAll(homedir.MustExpand(SupportDir))
+		}
```

Comment:
return a func to do the supportdir cleanup, so running `puma-dev -uninstall` will correctly leave you in a good state to `-install` again. 

See https://github.com/puma/puma-dev/pull/246/files#diff-6da8dfebc5fea29023190a0d10ab838dR62 as to why this is funky but necessary at the moment.